bin/test-lxd-vm

```diff
@@ -0,0 +1,152 @@
+#!/bin/sh
+set -eux
+
+cleanup() {
+    echo ""
+    if [ "${FAIL}" = "1" ]; then
+        echo "Test failed"
+        exit 1
+    fi
+
+    echo "Test passed"
+    exit 0
+}
+
+poolDriverList="${1:-dir btrfs lvm zfs}"
```

Comment:
Any reason not to do ceph too?

```
curl -s http://canonical-lxd.stgraber.org/config/ceph.sh | sh
```

Should get you the credentials in place for it.

Reply:
@stgraber ah ok I'll provision lantea and try it with that.